test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -31,20 +31,22 @@
 
 (deftest ^:integration legacy-routes
   (testing "The legacy web routing service properly handles old routes."
-    (bootstrap/with-puppetserver-running app
-      {}
-      (is (= 200 (:status (http-get "/v2.0/environments"))))
-      (is (= 200 (:status (http-get "/production/node/localhost"))))
-      (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
+    (bootstrap/with-puppetserver-running-with-mock-jrubies
+     app
+     {}
+     (is (= 200 (:status (http-get "/v2.0/environments"))))
+     (is (= 200 (:status (http-get "/production/node/localhost"))))
+     (is (= 200 (:status (http-get "/production/certificate_statuses/all")))))))
```

Comment:
This seems like one of the tests that is probably relying on the mocked `handleRequest`.  It kind of feels like we are losing some of the value of this test by mocking that - now we're really only validating that `handleRequest` gets called, not that it gets called in a manner that would actually yield a valid result from the ruby code... right?

I could see a case being made here that these tests should potentially be making some additional assertions on the body of the response, and that the lack of that coverage is what made it seem acceptable to just mock 200s coming back, but I'm iffy on this one.